Gemfile

```diff
@@ -151,6 +149,9 @@ group :development do
   gem 'capistrano-rails', '~> 1.1'
   gem 'capistrano-passenger'
 
+  # Improved syntax for creating cron jobs
+  gem 'whenever', '~> 0.9', require: false
```

Comment:
This was already in the Gemfile. I fixed the dupe and pegged to a version also. You are right to have it in the dev only block though I think :)